newIDE/app/src/UI/Paper.js

```diff
@@ -27,31 +27,43 @@ export const getBackgroundColor = (
     ? gdevelopTheme.paper.backgroundColor.medium
     : gdevelopTheme.paper.backgroundColor.light;
 
-const Paper = ({
-  id,
-  children,
-  background,
-  elevation,
-  variant,
-  style,
-  square,
-}: Props) => {
-  const gdevelopTheme = React.useContext(GDevelopThemeContext);
-  const backgroundColor = getBackgroundColor(gdevelopTheme, background);
-  return (
-    <MuiPaper
-      id={id}
-      variant={variant}
-      elevation={elevation || 0}
-      style={{
-        backgroundColor,
-        ...style,
-      }}
-      square={!!square}
-    >
-      {children}
-    </MuiPaper>
-  );
-};
+export type PaperInterface = {|
+  scrollToBottom: (options: ?{| smooth?: boolean |}) => void,
+|};
+
+const Paper = React.forwardRef<Props, PaperInterface>(
+  ({ id, children, background, elevation, variant, style, square }, ref) => {
+    const gdevelopTheme = React.useContext(GDevelopThemeContext);
+    const backgroundColor = getBackgroundColor(gdevelopTheme, background);
+    const paperRef = React.useRef<?MuiPaper>(null);
+
+    const scrollToBottom = React.useCallback(options => {
+      if (paperRef.current) {
+        paperRef.current.scroll({
+          top: 1000000,
```

Comment:
that seems random 😅  can't scroll to bottom: 0 ?
Maybe a quick comment if this is a hack

Reply:
It is indeed random. And it also shows a deeper issue, I'll try to fix it but I'm not sure I can do it easily

Reply:
Fixed that, removed the ref entirely